desktop/use-desktop/container.md

```diff
@@ -48,13 +41,31 @@ The **container view** displays **Logs**, **Inspect**, **Terminal**, **Files**,
     - Use the **Clear terminal** icon in the top right-hand corner to clear the logs terminal. 
     - Select and view external links that may be in your logs. 
 
-- Select **Inspect** to view low-level information about the container. You can see the local path, version number of the image, SHA-256, port mapping, and other details.
+### Inspect
+
+- Select **Inspect** to view low-level information about the container. It displays the local path, version number of the image, SHA-256, port mapping, and other details.
+
+### Integrated terminal
+
+From the **Terminal** tab, lets you use the integrated terminal, on a running container, directly within Docker Desktop. You are able to quickly run commands within your container so you can understand its current state or debug when something goes wrong.
+
+Using the integrated terminal is the same as running `docker exec -it <container-id> /bin/sh`, or `docker exec -it <container-id> cmd.exe` if you are using Windows containers, in your external terminal. It also:
+
+- Automatically detects the default user for a running container from the image's Dockerfile. If no use is specified it defaults to `root`.
```

Comment:
Possible typo? Wasn't sure if this was meant to be:
```suggestion
- Automatically detects the default user for a running container from the image's Dockerfile. If no user is specified it defaults to `root`.
```

Reply:
Oops again! Yes meant to be 'user'!